.azure-pipelines/buildAndPackage.yml

```
@@ -22,15 +20,58 @@ trigger:
       - THIRD PARTY NOTICES
       - README.md
 
-pr: none
+pr: 
+- dev
 
 pool:
-  vmImage: ubuntu-latest
+  vmImage: windows-latest
 
-steps: 
-- template: templates/secure-files.yml
-- template: templates/build-and-coverage.yml
-- template: templates/publish-artifacts.yml
-- template: templates/alert-failure.yml
+stages:
+- stage: Build
+  jobs:
+  - job: Build
+    steps: 
+    - template: templates/build/checkout-and-credscan.yml
+    - template: templates/build/install-java.yml
+    - template: templates/build/secure-files.yml
+    - template: templates/build/build-and-coverage.yml
+    - template: templates/build/publish-artifacts.yml
+    - template: templates/alert-failure.yml
 
+- stage: Maven-Preview
+  condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/dev'))
+  jobs: 
+  - job: Maven-Preview
+    steps: 
+    - template: templates/download-artifacts.yml
+    - template: templates/maven-preview/secure-files.yml
+    - template: templates/maven-preview/copy-and-build.yml
+    - template: templates/alert-failure.yml
 
+- stage: Maven-Release-And-Github-Release
+  dependsOn: []
+  condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
```

```suggestion
  condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/dev'))
```
for this repo, we don't have a master branch, switch to dev